components/script/dom/webglrenderingcontext.rs

```diff
@@ -1157,6 +1157,16 @@ impl WebGLRenderingContextMethods for WebGLRenderingContext {
             .unwrap()
     }
 
+    // https://www.khronos.org/registry/webgl/specs/latest/1.0/#5.14.9
+    fn IsShader(&self, shader: Option<&WebGLShader>) -> bool {
+        shader.map_or(false, |s| !s.get_is_deleted())
```

Comment:
I can not tell from the spec if I should be checking for more then just the is_deleted field here.  The compilation status seems like it should also be checked here as well.